site/en/blog/pop-ups-theyre-making-a-resurgence/index.md

```diff
@@ -0,0 +1,901 @@
+- When it appears, it gets promoted to the top layer. That means it gets promoted above the `document` in the viewport. You don’t have to manage `z-index` or worry about where your pop-up is in the DOM. It could be deep down nested in the DOM, with clipping ancestors. You can also see which elements are currently in the top layer through DevTools. For more on the top layer, [check out this article](/blog/what-is-the-top-layer/).
+
+![GIF of DevTools top layer support being demonstrated](https://wd.imgix.net/image/1D9D0Ls1ATa2ZPA9x2ZWrGFyZzT2/36Yck7O77zDipSNGNNbB.gif?auto=format&w=1600
+)
+
+- You get "Light Dismiss" out of the box. By that, we mean you can close the pop-up with a close signal, such as clicking outside the pop-up, keyboard-navigating to another element, or pressing the **Esc** key. Open it up again and try it out!
+
+What else do you get with pop-up? Let's take the example further. Consider this demo with some content on the page.
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'GRxLZYe',
+    height: 450,
+    tab: 'result'
+  }
+%}
+
+{% Video {
+    controls: true,
+    src: "video/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/aZtRNylbFkLDkLr3PiLs.mp4"
+  }
+%}
+
+That floating action button has fixed positioning with a high `z-index`.
+
+```css
+.fab {
+  position: fixed;
+  z-index: 99999;
+}
+```
+
+{% Aside %}
+The floating action button could also be a pop-up. But, here we are showing off the power of having access to the top layer. More on floating action buttons as pop-ups later.
+{% endAside %}
+
+The pop-up content is nested in the DOM, but when you open the pop-up, it gets promoted above that fixed position element. You don’t need to set any styles.
+
+You may also notice that the pop-up now has a `::backdrop` pseudo-element. All elements that are in the top layer get a styleable `::backdrop` pseudo-element. This example  styles `::backdrop` with a reduced alpha background color and a backdrop filter, which  blurs out the underlying content.
+
+{% Aside %}
+By default, the `::backdrop` on a pop-up has `pointer-events: none` set.
+{% endAside %}
+
+
+## Styling a pop-up
+
+Let's turn our attention to styling the pop-up. By default, a pop-up has a fixed position and some applied padding. It also has `display: none`. You could override this to show a pop-up. But, that wouldn't promote it to the top layer.
+
+```css
+[popup] { display: block; } 
+```
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'ZExZOpX',
+    height: 450,
+    tab: 'result'
+  }
+%}
+
+{% Video {
+    controls: true,
+    src: "video/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/u2i7C6POVKE67wdjoZBK.mp4"
+  }
+%}
+
+If you want a pop-up to be shown on load, you can use the `defaultopen` attribute.
+
+```html
+<div popup defaultopen id=”pop”>Pop-up content!</div>
+```
+
+{% Codepen {
+    user: 'web-dot-dev',
+    id: 'KKoYMmm',
+    height: 450,
+    tab: 'result'
+  }
+%}
+
+{% Video {
+    controls: true,
```

Comment:
Do you want these to autoplay and loop? You could do something like this for your videos:

```
{% Video src="video/Dyx9FwYgMyNqy1kMGx8Orz6q0qC3/IuseCtWYY3PzX1KJJRaZ.mp4", autoplay=true, controls=true, loop=true, muted=true %}
```